docs/conf/eu/2016/news/schedule.md

```diff
@@ -0,0 +1,85 @@
+:template: 2016/eu.html
+
+Announcing the conference schedule
+=====================================================================================
+
+Write the Docs is just over three weeks away now! We hope you are all as excited as we are for the conference.
+
+Tickets are **almost sold out**, so make sure to [purchase a ticket][ticket] now, if you haven't already. We'll be closing ticket sales on the *5th September* to leave us enough time to print shirts and badges -- don't miss out!
+
+[ticket]: https://ti.to/writethedocs/write-the-docs-eu-2016/
+
+We're putting the finishing touches on the conference, so this email
+contains more information for you about this year's events. We also hope you'll be able to join most if not all of the events outside the main conference talks, because they're some of our favorite parts of the event.
```

Comment:
this says "event" 3 times in different contexts. Should probably do like:

We're putting the finishing touches on the conference. Write the Docs is all about community, so we make sure you have a lot of time to connect with other documentarians. We hope you'll be able to join most if not all of the events outside the main conference talks, because they're some of our favorite parts of the event.